packages/itwin/ec3-widget/package.json

```diff
@@ -107,6 +107,7 @@
     "react-redux": "^7.2.2",
     "redux": "^4.1.0",
     "rimraf": "^3.0.2",
+    "sinon": "^17.0.1",
```

Comment:
Sinon should no longer be needed once you will add a way to override the localization (not just the individual strings)

Reply:
removed sinon

Reply:
Also, maybe it would be worth to revert the lock file.

Reply:
reverted